11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,39 @@
+.container{
+  display: flex;
+  flex-direction: column;
+
+}
+.box1{
+  background: #000 url("../img/bg2.jpg") no-repeat right;
+  background-size: cover;
+  height: 400px;
+  color: white;
+  font-size: 1em;
```

Comment:
Why overriding all of the font-sizes? It's okay to do so, just understand why. Also, the sizes you chose are probably too small; don't want to make people squint...